app/templates/components/events/view/overview/general-info.hbs

```diff
@@ -46,9 +46,9 @@
         <td><strong>{{t 'Session types'}}</strong></td>
         <td>
           {{#if this.data.sessionTypes}}
-            {{#each this.data.sessionTypes as |sessionType index|}}
+            {{#each this.data.sessionTypes as |sessionType index|~}}
               {{if (not-eq index 0) ','}} {{sessionType.name}}
-            {{/each}}
+            {{~/each}}
```

Comment:
What does this do?

Reply:
this removes unwanted space after each loop iteration

Reply:
https://discuss.emberjs.com/t/removing-whitespace-with-angle-bracket-syntax/17094/9